File: mathematics/right_angle_triangle.py
#Given 3 numbers A,B,C print 'yes' if they can form the sides of a right angled triangle,otherwise 'no'.
A,B,C=map(int,input().split())
c=A**2+B**2
d=C**2
if(c==d):
	print("yes")
else:
	print("no")

File: javascript/basics/right_angle_triangle.js
//Given 3 numbers A,B,C print 'yes' if they can form the sides of a right angled triangle,otherwise 'no'.
const readline = require('readline');

const inp = readline.createInterface({
  input: process.stdin
});

const m = [];

inp.on("line", (data) => {
	m.push(data);
});

inp.on("close", () => {
var	T=m[0].split(" ");
var A=(parseInt(T[0])),B=(parseInt(T[1])),C=(parseInt(T[2]));  
var c=A**2+B**2;
var d=C**2;
if(c==d)
{
	console.log("yes");
}
else
{
console.log("no");
}
});
